static/scss/style.scss

```diff
@@ -545,6 +558,22 @@ nav.revealed {
   margin-top: 15px;
 }
 
+.party-header.democrat {
```

Comment:
Give the `.party-header` a color so it will fall back when we don't recognize the party and remove the extra styles.

Reply:
gotcha, done